topcoffea/modules/histEFT.py

```diff
@@ -342,8 +342,9 @@ def make_scaling(self, wc_list=None):
                 step += 1
             else:
                 scaling[:,i] /= 2
-        return np.nan_to_num(scaling/np.expand_dims(scaling[:,0], 1), 0) #divide by sm
```

Comment:
Just curious, was this actually causing a crash, or was it a warning about `NaN` that `nan_to_num` then set to 0?

Reply:
This wasn't causing a crash, but rather raised a warning in empty bins. Explicitly, the warning read
```py
RuntimeWarning: invalid value encountered in divide
```
This change keeps the same functionality while getting rid of that warning for the case of any number empty bins.